### DISCUSSION TITLE
界面大小

### DISCUSSION THREAD

**Discussion by SACMaker** [Planshit/Tai#54] [Ideas]
每次打开软件,我都感觉界面太小很紧凑!
![image](https://user-images.githubusercontent.com/43346914/178762852-9d0e40d8-baf2-4412-9d82-07826ebb0a88.png)
我个人觉得这样默认界面大小比较合适
![image](https://user-images.githubusercontent.com/43346914/178763125-2c220513-5d01-4b43-9edd-8f8e2aab427f.png)

**Comment by noberumotto** [comment]
你的屏幕分辨率是？

**Comment by noberumotto** [comment]
已在[1.3.0.0](https://github.com/Planshit/Tai/releases/tag/1.3.0.0)版本中加入 `保留窗口大小` 功能，可以保存你的窗口大小，在下次打开窗口时会以保存的尺寸显示了。

**Reply by SACMaker** [reply]
NICE JOB!
